server/config/requestHandler.js

```diff
@@ -38,7 +38,7 @@ module.exports = {
   postSignin: function(req, res) {
     // check user exist in the database
     console.log('postSignin fired!', req.body);
-    var username = req.body.username.toLowerCase();
+    var username = req.body.username;
```

Comment:
@kevinwin so if a user had an uppercase in their username it would get passed to the server as uppercase? If so, we should probably convert to lower case otherwise we could have multiple identical username with different case... e.g., kevin, Kevin, etc.